db/migrations/20240318200010-update-exisiting-remediation-names.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+module.exports = {
+    up: async (q, {STRING}) => {
+        const transaction = await q.sequelize.transaction()
+        try {
+            //===================================================
+            // The next 2 queries will update existing remediation names to ensure that there are
+            // no duplicate remediation names within an organization
+            //===================================================
+
+            // Find duplicate remediation names within an organization and add the username to the beginning of the name
+            // Example: 'test1@redhat.com: duplicate name', 'test2@redhat.com: duplicate name', 'test3@redhat.com: duplicate name'...
+            const update_dups_in_org = `UPDATE remediations
+                                        SET name = created_by || ': ' || name 
```

Comment:
Why do we append username here?
Can this rather prepend creation/modification time in the unix format? That could resolve the duplicates within an account and would not require names with incrementing numbers.

Reply:
This can be pulled from `created_at`. Just check it's type. Looking at some migrations this might be just a date (and not datetime).

Reply:
The `created_at` type is `DATE` but it looks like the format is `2024-03-04T16:12:51.012Z` so I think this would work. I think one query like this would be enough to update all existing duplicate names: 
```
UPDATE remediations
SET name = created_at || ': ' || name 
WHERE (name, tenant_org_id) IN
(
           SELECT name, tenant_org_id
           FROM remediations
           GROUP BY name, tenant_org_id
           HAVING ( COUNT( name ) > 1 )
)
```

Reply:
Yes, looks like it should be a datetime, as per [docs](https://sequelize.org/api/v6/class/src/data-types.js~date). I'd transform that to unix time to get a number, and add it to the end (not to the start of the original name). @rexwhite thought?